### PR TITLE
Added some new examples

### PR DESCRIPTION
**Pull request by farisT**
- Added playgrounds for both languages
- Updated Constructor example

New examples:
- Objects
- Overwriting classes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +110,25 @@ const persons: string[] = ['John', 'William'];
 final List<String> persons = ['John', 'William'];
 ```
 
+### Objects
+---
+```ts
+const peopleObject = {
```
Comment (jeroenouw) [inline README.md]: `const person = {`

File: README.md [hunk 2]
```diff
@@ -108,6 +110,25 @@ const persons: string[] = ['John', 'William'];
...
+
+console.log(peopleObject.name)  // John
+```
+
+```dart
+class People {
+  String name = 'John';
+}
+
+var peopleObject = Persons();
```
Comment (jeroenouw) [inline README.md]: Should be `= People();`

File: README.md [hunk 3]
```diff
@@ -155,12 +176,35 @@ class Foo extends StatefulWidget  {}
...
+#### TypeScript
+```ts
+export class Foo {
+   constructor() {
+     super()
+   }
 }
 ```
+
 #### Dart/Flutter
```
Comment (jeroenouw) [inline README.md:205]: It's now Dart specific, so Flutter can be removed

File: README.md (hunk 2 above)
Comment (jeroenouw) [reply README.md]: Also `var person` here

File: examples/objects/objects.dart [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class People {
```
Comment (jeroenouw) [inline examples/objects/objects.dart]: Please update also the examples

File: README.md (hunk 3 above)
Comment (farisT) [reply README.md:205]: This can be done in Flutter as well.

**Review by jeroenouw — APPROVED** [review]

**Pull request merged**